MAP/src/components/Map/util.js

```diff
@@ -19,6 +19,25 @@ export function worldStyle(lockdown_status) {
   return value;
 }
 
+export function worldStyleColor(lockdown_status) {
+  let value;
+  switch (lockdown_status) {
+    case '1':
+      value = 'red'; // yes
+      break;
+    case '2':
+      value = 'orange'; // partial
+      break;
+    case '3':
+      value = 'green'; // no
```

Comment:
these colors should map to the existing styles. It seems like they are, I just want to be sure. 
```
    --ld-yellow-1: #f6b26b;
    --ld-green-1: #93c47d;
    --ld-red-1: #e06666;
```